pocs/core.py

```diff
@@ -295,7 +299,11 @@ def is_safe(self, no_warning=False):
             if no_warning is False:
                 self.logger.warning('Unsafe conditions: {}'.format(is_safe_values))
 
-            # Not safe so park unless we are not active
+            # Not safe, so close the dome and park the mount.
+            if not self.observatory.close_dome():
+                self.logger.critical('Unable to close dome!')
+
+            # TODO(jamessynge): Determine why parking is conditional. Is self.park() idempotent?
```

Comment:
If you try to `park` from a state that doesn't support the transition it will complain. The `transitions` library has since added support for nested states as well as other features that could clean this up, but we should discuss in another PR.

Reply:
Got it. I've been wondering about having separate state machines for the different equipment, since it doesn't make sense to continue expanding the base state machine.